internal/tjson/tjson_test.go

```diff
@@ -222,28 +229,38 @@ func testJSON(t *testing.T, testCases []testCase, newFunc func() tjsontype) {
 
 func fuzzJSON(f *testing.F, testCases []testCase) {
 	for _, tc := range testCases {
-		f.Add(tc.j)
+		schema, err := tc.schema.Marshal()
+		if err != nil {
+			f.Fatalf("failed to marshal schema: %v", err)
+		}
+
+		f.Add(tc.j, string(schema))
+
 		if tc.canonJ != "" {
-			f.Add(tc.canonJ)
+			f.Add(tc.canonJ, string(schema))
 		}
 	}
 
-	// TODO Add fuzzing for schema https://github.com/FerretDB/FerretDB/issues/943
-	schema := testCases[0].schema
-
-	f.Fuzz(func(t *testing.T, j string) {
+	f.Fuzz(func(t *testing.T, j, schema string) {
 		t.Parallel()
 
 		// raw "null" should never reach UnmarshalJSON due to the way encoding/json works
 		if j == "null" {
 			t.Skip()
 		}
 
+		s := newSchema()
+
+		err := s.Unmarshal([]byte(schema))
+		if err != nil {
+			t.Skip()
```

Comment:
This skip is for invalid schema only right?

Reply:
Seems so